wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -47,14 +47,72 @@ When adding redirects for a new or existing article, keep in mind that they shou
+
+#### Bypassing CI checks
+
+CI checks ensure that changes to the wiki keep it in a consistent state, but ways to bypass them exist as a fail safe, such that pull requests can still be merged in the event of false errors or discovered bugs in the checks. There are a few situations where intentionally bypassing a CI check is acceptable, as outlined below. Contact a [maintainer](/wiki/osu!_wiki/Maintenance/List_of_maintainers) if you need to bypass a check for a reason not mentioned here.
+
+For reference, below is a table of all CI checks in order:
+
+| # | Check | Tool | Explanation | Bypass |
+| :-: | :-- | :-- | :-- | :-- |
+| 1 | File sizes | [`scripts/ci/inspect_file_sizes.sh`](https://github.com/ppy/osu-wiki/blob/master/scripts/ci/inspect_file_sizes.sh) | Whether an image file is below the [news post and wiki article image file size limit](/wiki/Article_styling_criteria/Formatting#file-size) (1 MB). Gives a warning for files above 0.5 MB. | None. |
+| 2 | Markdown | [remark](https://github.com/remarkjs/remark) via [`scripts/ci/run_remark.sh`](https://github.com/ppy/osu-wiki/blob/master/scripts/ci/run_remark.sh) | Whether Markdown syntax is correct and consistent in wiki articles and news posts. | Add `<!-- lint ignore rule-name -->` above the offending line, where `rule-name` is the rule to ignore. |
+| 3 | YAML | [yamllint](https://github.com/adrienverge/yamllint) via [`scripts/ci/run_yamllint.py`](https://github.com/ppy/osu-wiki/blob/master/scripts/ci/run_yamllint.py) | Whether YAML syntax is correct and consistent in the [`redirect.yaml`](https://github.com/ppy/osu-wiki/blob/master/wiki/redirect.yaml) file and in [front matter](/wiki/Article_styling_criteria/Formatting#front-matter) | None. |
+| 4 | Broken wiki links | `check-links` command of [`osu-wiki-tools`](https://github.com/Walavouchey/osu-wiki-tools) | Whether internal [wiki links](/wiki/Article_styling_criteria/Formatting#wiki-links) point to an actual article, news post (for news post links), or section thereof. | Add `SKIP_WIKILINK_CHECK` anywhere in the pull request description. |
+| 5 | Outdated translations | `check-outdated-articles` command of [`osu-wiki-tools`](https://github.com/Walavouchey/osu-wiki-tools) | Whether translations are properly [marked as outdated](/wiki/Article_styling_criteria/Formatting#outdated-translations) when updating an English article. | Add `SKIP_OUTDATED_CHECK` anywhere in the pull request description. |
+
+##### Markdown [`no-heading-punctuation`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-heading-punctuation) remark lint rule
+
+Trailing periods in headings normally do not occur because headings are generally not sentences, and are therefore disallowed. Sometimes trailing punctuation is needed because e.g. a song title in the heading contains it, or a sentence is put into a heading in an FAQ section.
```

Comment:
"or a sentence is put into a heading" is I think the reason this check exists, so no. if it "is put into a heading", the trailing period still has to go
```suggestion
Trailing periods in headings normally do not occur because headings are generally not sentences, and are therefore disallowed. Sometimes trailing punctuation is needed because e.g. a song title in the heading contains it.
```

Reply:
for full sentences, that's basically the only place you'd use it, along with trailing questions marks. i'm kind of referring to [this currently sole example](https://github.com/ppy/osu-wiki/blob/1478755324ad093078d29949579b286f7b232384/wiki/Help_centre/Upgrading_to_lazer/en.md?plain=1#L150-L152) (in all other faq sections, the headings end in `?` or `!`)

```suggestion
Trailing periods in headings normally do not occur because headings are generally not sentences, and are therefore disallowed. Sometimes trailing punctuation may be needed because e.g. a song title or artist name contains it, or for an FAQ-style section.
```

Reply:
hmm i'll leave it like this then

```suggestion
Trailing periods in headings normally do not occur because headings are generally not sentences, and are therefore disallowed. However, sometimes trailing punctuation may be needed because e.g. a song title or artist name contains it.
```